samples/boards/kv260_starter_kit_rpu/vitis/debug_zephyr.tcl

```diff
@@ -0,0 +1,30 @@
+# Usage with xsct:
+# 
+# source debug_zephyr.tcl
+# 
+connect -url tcp:127.0.0.1:3121
+source /tools/Xilinx/Vitis/2022.1/scripts/vitis/util/zynqmp_utils.tcl
```

Comment:
Avoid hard-coded `/tools/Xilinx` path

Reply:
the tools/Xilinx path is always selected from Vitis during installation, of course the 2022.1 part from its path should be handled to avoid problems between specific versions.